packages/react/src/components/ExpandCollapsePanel/index.css

```diff
@@ -1,5 +1,35 @@
+:root {
+  --expandcollapse-trigger-background-color: transparent;
+  --expandcollapse-trigger-color: var(--gray-90);
+  --expandcollapse-panel-background-color: transparent;
+  --expandcollapse-panel-color: var(--gray-90);
+}
+
 .ExpandCollapse__trigger {
   font-size: inherit;
   display: flex;
   align-items: center;
+  background-color: var(--expandcollapse-trigger-background-color);
+  color: var(--expandcollapse-trigger-color);
+}
+
+.fullWidth {
+  width: 100%;
+}
+
+.ExpandCollapse__trigger-title {
+  flex-grow: 1;
+  text-align: left;
+}
+
+.ExpandCollapse__panel {
+  padding: var(--space-smallest) 0;
```

Comment:
Noticed this one thing, the padding mucks around with smooth transitions creating some jank so let's remove this for now.

```suggestion
```